main.go

```diff
@@ -1,22 +1,29 @@
 package main
 
 import (
+	"encoding/json"
 	"fmt"
 	"os"
 	"path/filepath"
 	"strings"
 
 	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/errorutil"
+	"github.com/bitrise-io/go-utils/fileutil"
 	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-utils/pathutil"
 	"github.com/bitrise-tools/go-steputils/stepconf"
+	"github.com/bitrise-tools/xcode-project/serialized"
 )
 
 // Config ...
 type Config struct {
-	ProjectPath    string          `env:"project_path,dir"`
-	ExpoCLIVersion string          `env:"expo_cli_verson,required"`
-	UserName       string          `env:"user_name"`
-	Password       stepconf.Secret `env:"password"`
+	Workdir                 string          `env:"workdir"`
```

Comment:
Why is it changed from `ProjectPath`? Without modifying this field it could be a non-breaking change.

Reply:
`env:"workdir"` -> is it missing the `dir` tag?